doozer/doozerlib/rhcos.py

```
@@ -233,8 +233,12 @@ def rhel_build_meta_layered(self, pullspec: str):
             rhel_minor = build_id.split(".")[1]
             url = f"{RHCOS_RELEASES_STREAM_URL}/rhel-{rhel_major}.{rhel_minor}/builds/{build_id}/{self.brew_arch}/commitmeta.json"
             logger.info(f"Send request to {url}")
-            with request.urlopen(url, timeout=60) as req:
-                return json.loads(req.read().decode())['rpmostree.rpmdb.pkglist']
+            try:
+                with request.urlopen(url, timeout=60) as req:
```

Ruff (0.15.13)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)